### PR TITLE
Switch from manual Fastly deployments to automated deployments

### PR DESCRIPTION
**Pull request by JakeChampion**
### Problem
- Deployments for our VCL are currently manually via the Fastly GUI.
- We do not have a qa/staging environment for our VCL.
- Our VCL is not tested.
### Solution
- [x] Add VCL to our codebase and deploy with the [fastly-tools](https://github.com/Financial-Times/fastly-tools) CLI.
- [x] Create a qa/staging environment for our VCL, http://origami-build-qa.ft.com/v2/ *
- [x] Create a ci environment https://ci.build.origami.ft.com/ \* and run end-to-end tests on this as well as on qa/staging environment before promotion occurs.
- [x] Document the environment variables required to deploy VCL to production.

I would have loved to be able to run the `varnishd` compiler and `varnishtest` applications against our VCL but we are using Fastly's old and heavily customised Varnish (version 2.1.5). If you try and run the compiler against our `log.vcl` file, you will see an error about the `randombool` function which is a Fastly-specific function.

<details><summary>

Example output of running `varnishd` against `log.vcl`</summary>

```
❯ varnishd -C -f log.vcl
Message from VCC-compiler:
Syntax error in condition, expected '(', '!' or variable name, found 'randombool'
(input Line 3 Pos 7)
  if (randombool(1, 4)) {
------##########---------
Running VCC-compiler failed, exit 1
```

</details>

happy to bike-shed the sub-domains. *

### CODE REVIEW

**Comment by JakeChampion** [comment]
Thanks to @seanmtracey , we now have http://build-qa.origami.ft.com/v2/ :-)

**Comment by triblondon** [comment]
I thought you were switching to direct subdomains of ft.com, for TLS related reasons?

**Comment by JakeChampion** [comment]
@triblondon Yep, we noticed my mistake and now have http://origami-build-qa.ft.com/v2/

**Comment by JakeChampion** [comment]
The VCL has been pushed to http://origami-build-qa.ft.com/v2/ and activated.

**Comment by rowanmanning** [comment]
From what I can see, this looks great! @JakeChampion, fancy talking through it at some point today?

**Comment by JakeChampion** [comment]
This PR is waiting on https://origami-build-ci.ft.com existing as a domain.

File: lib/utils/hostnames.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ exports.known = [
 	'buildservice-fastly.ft.com',
 	'buildservice.ft.com',
 	'origami-build.ft.com',
+  'origami-build-qa.ft.com',
+  'origami-build-ci.ft.com',
```
Comment (rowanmanning) [inline lib/utils/hostnames.js]: Indentation is off here

**Pull request merged**